### PR TITLE
Endpoints for saving Owner and finding Owner by Id

### PR DESCRIPTION
**Pull request by MartinZhelyazkov**
Endpoints for saving Owner and finding Owner by Id

### CODE REVIEW

File: src/main/java/com/example/spring_into/controller/OwnerController.java [hunk 1]
```diff
@@ -23,4 +26,13 @@ Owner updateOwner(@RequestBody OwnerRequest request,
     void deleteOwner(@PathVariable("ownerId") Long ownerId) {
         ownerService.deleteOwnerById(ownerId);
     }
+    @PostMapping
```
Comment (ivanvolkoff) [inline src/main/java/com/example/spring_into/controller/OwnerController.java:30]: empty line to separate methods

File: src/main/java/com/example/spring_into/controller/OwnerController.java [hunk 2]
```diff
@@ -23,4 +26,13 @@ Owner updateOwner(@RequestBody OwnerRequest request,
...
+    ResponseEntity<String> addOwner(@RequestBody OwnerRequest ownerRequest){
+        ownerService.addOwner(ownerRequest);
+        return null;
+    }
```
Comment (ivanvolkoff) [inline src/main/java/com/example/spring_into/controller/OwnerController.java:34]: same as above

File: src/main/java/com/example/spring_into/repository/OwnerRepository.java [hunk 3]
```diff
@@ -10,4 +10,6 @@
 public interface OwnerRepository extends JpaRepository<Owner, Long> {
 
     Optional<Owner> findByEmail(String email);
+
```
Comment (ivanvolkoff) [inline src/main/java/com/example/spring_into/repository/OwnerRepository.java]: delete empty lines

File: src/main/resources/application.yml [hunk 4]
```diff
@@ -5,7 +5,7 @@ spring:
     url: jdbc:mysql://localhost:3306/toll_system?createDatabaseIfNotExist=true
     driverClassName: com.mysql.cj.jdbc.Driver
     username: root
-  #    password: <your password>
+    password: admin
```
Comment (ivanvolkoff) [inline src/main/resources/application.yml]: exposed credentials

**Review by ivanvolkoff — CHANGES_REQUESTED** [review]
changes should be implemented

**Review by ivanvolkoff — APPROVED** [review]

**Review by Oreshcode — APPROVED** [review]

**Pull request merged**